BlueShift-iOS-Extension-SDK/BlueShiftPushAnalytics.m

```diff
@@ -64,7 +64,8 @@ + (void) getRequestWithURL:(NSString *)urlString andParams:(NSDictionary *)param
     NSURLSession *defaultSession = [NSURLSession sessionWithConfiguration: sessionConfiguraion delegate: nil delegateQueue: [NSOperationQueue mainQueue]];
     
     NSString *paramsString = [[NSString alloc] init];
-    for(id key in params) {
+    NSArray *keys = [[params allKeys] sortedArrayUsingSelector:@selector(caseInsensitiveCompare:)];
```

Comment:
Is it any feature change..?

Reply:
The get url params were getting generated in random order, so this way all get params will have a fixed order.